src/Compilers/CSharp/Test/Emit2/Semantics/ParamsCollectionTests.cs

```diff
@@ -14319,6 +14319,154 @@ static void Test(params CollectionBuilderAttribute a)
                 );
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/74734")]
+        public void Cycle_16()
+        {
+            if (CompilationExtensions.EnableVerifyIOperation &&
+                ExecutionConditionUtil.Configuration == ExecutionConfiguration.Debug)
+            {
+                // A debug assert fails but an invalid operation is correctly created otherwise.
```

Comment:
Do we have a work item tracking this?

Reply:
No, I will create one, thanks.

Reply:
On a second thought, maybe it's better to just remove the assert.